src/components/timecard/select-timecard-period-type/SelectTimecardPeriodType.jsx

```diff
@@ -24,6 +16,17 @@ const SelectTimecardPeriodType = () => {
     shouldFocusError: false,
   });
 
+  const [timePeriods, setTimePeriods] = useState([]);
+  const getResponse = async () => {
+    const response = await getTimePeriodTypes({
+      tenantId: '00000000-0000-0000-0000-000000000000',
+    });
+    setTimePeriods(response.data.items.map((item) => item.name));
+  };
+  useEffect(() => {
+    getResponse();
+  }, []);
```

Comment:
This will get the data on every render, which I think is fine for now. Eventually we can look at getting this 'reference data' only once when the application starts and passing the data down to the component

Reply:
This is related to the conversation above re: default options